CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - Select initial/default value of Radio Button on client info screen
 
+### Changed
+- Change font size and spacing on Patient list screen as per the latest design
```

Comment:
`- Reduce font size and spacing on patient list screen`

*".. as per latest design"* would be relative depending on the time the changelog is checked